lib/ansible/module_utils/aws/core.py

```diff
@@ -263,6 +295,14 @@ def __getattr__(self, name):
             return unwrapped
 
 
+def inject_botocore_logged_actions(method):
```

Comment:
What is this method used for? I don't see any references to it.

Reply:
Oh, good catch. I was going to decorate exit_json, fail_json and fail_json_aws, decided against it midway through and then forgot to remove it.